Makefile

```diff
@@ -2,7 +2,8 @@
 RM := rm -f
 SSH_USER := core
 TERRAFORM_INSTALLER_URL := github.com/dcos/terraform-dcos
-DCOS_VERSION := 1.11
+DCOS_VERSION := 1.12
+CUSTOM_DCOS_DOWNLOAD_PATH := https://downloads.dcos.io/dcos/testing/1.12.0-beta1/dcos_generate_config.sh
```

Comment:
Why do we set `DCOS_VERSION` above and use the hardcoded value of `testing/1.12.0-beta1` here?

Reply:
Also, this should probably be `1.12.0-rc2` by now.

Reply:
This is going to changed post GA, so i left it as is until we are ready, i meant to put a label indicating as much but didnt find one.

Reply:
Better we update it so we can test in the meantime.